.github/workflows/build-test.yml

```
@@ -0,0 +1,30 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build-test:
+    name: Build and Test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: yarn --frozen-lockfile
+    - run: yarn buildSample
+    - run: yarn test
```

There is a separate script meant for testing on ci:

```suggestion
    - run: yarn ci
```

It produces better output for CI (no progress bar).

---

Sweet, updated to use this